scripts/benchmark/utils/chain_plumbing.py

```diff
@@ -81,19 +93,27 @@
 GenesisState = Iterable[Tuple[Address, Dict[str, Any]]]
 
 
-def get_chain(vm: Type[BaseVM], genesis_state: GenesisState) -> MiningChain:
+def get_chain(vm: Type[BaseVM], chain_db: BaseAtomicDB, genesis_state: GenesisState) -> MiningChain:
 
     chain = build(
         MiningChain,
         fork_at(vm, constants.GENESIS_BLOCK_NUMBER),
         disable_pow_check(),
-        genesis(params=GENESIS_PARAMS, state=genesis_state)
+        genesis(db=chain_db, params=GENESIS_PARAMS, state=genesis_state)
     )
 
     return chain
 
 
 def get_all_chains(genesis_state: GenesisState=DEFAULT_GENESIS_STATE) -> Iterable[MiningChain]:
     for vm in ALL_VM:
-        chain = get_chain(vm, genesis_state)
-        yield chain
+        # Chain using MemoryDB
+        memory_db_obj = AtomicDB()
+        memory_db_chain = get_chain(vm, memory_db_obj, genesis_state)
+        yield memory_db_chain
+
+        # Chain using LevelDB
+        with tempfile.TemporaryDirectory() as temp_dir:
```

Comment:
Looking at the CI run, this doesn't increase the time these benchmarks take as much as I had feared. With that in mind, I'd say we should default to run all tests using `LevelDB` and simplify the logic here. Basically, I think `get_all_chains` can roll back to what it looked like before and the `LevelDB` setup code can move into `get_chain`.